### PR TITLE
make mdl capable to be rendered server-side

### PR DESCRIPTION
**Pull request by digitalkaoz**
this PR allows an application using mdl to render on the server (e.g. react)

### CODE REVIEW

**Comment by digitalkaoz** [comment]
I signed it!

File: src/button/button.js [hunk 1]
```diff
@@ -113,10 +116,12 @@
 
   // The component registers itself. It can assume componentHandler is available
```
Comment (ZanderBrown) [inline src/button/button.js]: this comment becomes irrelavent

Comment (digitalkaoz) [reply src/button/button.js]: correct, removed it

**Comment by Garbee** [comment]
Server-side rendering was never a goal of the library when built. Material Components for Web is encouraged for dynamic applications and using SSR.

As MDL is no longer maintained, closing this out since there won't be any solid QA either pre-merge or any further patches past the next version.

Thank you for your time here, it is appreciated.

**Pull request closed without merging**